tools/android/MASTG-TOOL-0103.md

```diff
@@ -0,0 +1,19 @@
+---
+title: uber-apk-signer
+platform: android
+source: https://github.com/patrickfav/uber-apk-signer
+---
+
+A tool that helps to sign, [zip aligning](https://developer.android.com/studio/command-line/zipalign.html) and verifying multiple Android application packages (APKs) with either debug or provided release certificates (or multiple). It supports v1, v2, [v3](https://source.android.com/security/apksigning/v3) and [v4](https://source.android.com/security/apksigning/v4) Android signing schemes. Easy and convenient debug signing with embedded debug keystore. Automatically verifies signature and zipalign after every signing.
+
+Download the jar from the [latest release](https://github.com/patrickfav/uber-apk-signer/releases/latest) and run:
+
+```bash
+java -jar uber-apk-signer.jar --apks /path/to/apks
```

Comment:
```suggestion
$ java -jar uber-apk-signer.jar --apks </path/to/apks>
```
(I noticed this in https://mas.owasp.org/contributing/5_Style_Guide/#8-code-and-shell-commands )